.github/workflows/update-contribs.yml

```
@@ -0,0 +1,44 @@
+name: Update Contributors
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+    run-meta:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Check out the code
+          uses: actions/checkout@v3
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: '3.10'
+        - name: Upgrade pip
+          run: |
+            # install pip=>20.1 to use "pip cache dir"
+            python3 -m pip install --upgrade pip
```

```suggestion
            python -m pip install --upgrade pip wheel
```

`setup-python` links the Python version you specify to the `python` command, so you can omit the `3`. Also, may as well install/upgrade `wheel` here too, methinks.

---

ok thanks! does the core wheel have issues bugs ? or is this like upgrading pip just to ensure you have a current version?

---

AFAIK it's not installed by default in venvs or some default environments at all, and its good to always be using the latest version when testing here.